common/src/lib.rs

```diff
@@ -52,6 +61,19 @@ pub const GAS_VALUE_PREFIX: &[u8] = b"g::gas_tree";
 
 pub type ExitCode = i32;
 
+#[derive(Clone, Debug, Decode, Encode, PartialEq, TypeInfo)]
+pub struct Dispatch {
+    pub kind: DispatchKind,
+    pub message: Message,
```

Comment:
Considering about placing everywhere in Dispatch enum with variants of Native or "Common" messages

Reply:
Discussed, next PR